packages/gatsby-theme-apollo-core/src/utils/colors.js

```diff
@@ -3,8 +3,8 @@ const {colors} = require('@apollo/space-kit/colors');
 exports.colors = {
   primary: colors.indigo.dark,
   primaryLight: colors.indigo.lighter,
-  secondary: colors.pink.base,
-  tertiary: colors.teal.dark,
+  secondary: colors.pink.darker,
+  tertiary: colors.teal.darker,
```

Comment:
Looks awesome! Thanks @StephenBarlow. Since we only step from `dark` to `darker` on the second line, should we step up one time on the first as well, from `base` to `dark` instead of `darker`?

Reply:
Good Q! Based on @ana-demags's findings, these are the least-dark shades of each brand color that fit the contrast requirements for accessibility. So I believe we should stick with these updates.

Reply:
Sounds good to me!